db/app.js

```diff
@@ -1,17 +1,25 @@
 const express = require("express");
 const app = express();
 
-const { viewCategories, getMessage } = require("./controllers/app.controller");
+const {
+  viewCategories,
+  getMessage,
+  viewReviews,
+} = require("./controllers/app.controller");
 
 app.use(express.json());
 
 app.get("/api", getMessage);
 
+app.get("/not-a-route");
+
```

Comment:
For catching the wrong path plese try to make use of app.all('*',()={}).

https://www.geeksforgeeks.org/express-js-app-all-function/